src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -5,8 +5,10 @@ Release History
 
 2.3.1
 +++++
+* `az aks create` now defaults to Standard_DS2_v2 VMs.
```

Comment:
You need to bump the version as 2.3.1 just released.

Reply:
Done, thanks!

Reply:
As well as in the `setup.py`.

Reply:
Done.